calculator22.py

```diff
@@ -1,26 +1,5 @@
-operations_count = 0
-
-def main():
-    ask_again = True
-    while(ask_again):
-        a = input("Enter the numerator: ")
-        b = input("Enter the denominator: ")
-        result = perform_division(a,b)
-        print(result)
-        ask_again = input("Do you want to perform another operation? Enter yes or no: ")
-        if(ask_again == 'yes'):
-            ask_again = True
-        else:
-            ask_again = False
-            print("You performed " + str(operations_count) + " operations, bye!")
-
-
-def perform_division(a,b):
-    global operations_count
-    try:
-        operations_count += 1
-        return int(a)/int(b)
-    except Exception as e:
-        pass
-
-main()
\ No newline at end of file
+def set_user_noncompliant():
+    import os
+    root = 0
+    # Noncompliant: the process user is set to root.
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This method is running with elevated privileges. Privilege elevation occurs when you access resources with an extended set of permissions. When unattended, these privileges might allow attackers to compromise the system. Do not use elevated privileges when they can be avoided.

[Learn more](https://cwe.mitre.org/data/definitions/269.html)